openvino2tensorflow/openvino2tensorflow.py

```diff
@@ -67,6 +67,7 @@ def convert(
     output_h5,
     output_weight_and_json,
     output_pb,
+    dont_save_keras_model,
```

Comment:
To be precise, `dont_convert_from_keras_model` is more correct in meaning than `dont_save_keras_model`, since we are not outputting keras_model, but only reading keras model objects.